### DISCUSSION TITLE
Getting telethon.errors.rpcerrorlist.ChatAdminRequiredError

### DISCUSSION THREAD

**Discussion by Chappie18** [aahnik/tgcf#102] [Q&A] [answered]
```
ERROR:root:Chat admin privileges are required to do that in the specified chat (for example, to send a message in a channel which is not yours), or invalid permissions used for the channel or group (caused by SendMessageRequest)
Traceback (most recent call last):
  File "/data/data/com.termux/files/usr/lib/python3.9/site-packages/tgcf/past.py", line 32, in forward_job
    await send_message(client, destination, message)
  File "/data/data/com.termux/files/usr/lib/python3.9/site-packages/tgcf/utils.py", line 8, in send_message
    return await client.send_message(*args)
  File "/data/data/com.termux/files/usr/lib/python3.9/site-packages/telethon/client/messages.py", line 826, in send_message
    result = await self(request)
  File "/data/data/com.termux/files/usr/lib/python3.9/site-packages/telethon/client/users.py", line 30, in __call__
    return await self._call(self._sender, request, ordered=ordered)
  File "/data/data/com.termux/files/usr/lib/python3.9/site-packages/telethon/client/users.py", line 79, in _call
    result = await future
telethon.errors.rpcerrorlist.ChatAdminRequiredError: Chat admin privileges are required to do that in the specified chat (for example, to send a message in a channel which is not yours), or invalid permissions used for the channel or group (caused by SendMessageRequest)
```

I am trying to copy all the contents from a joined private channel to my own created channel as a backup.
But seems like script doesn't support copying contents from joined private channels, where i am just a user.
Is there anyway to make it possible?

using on Termux

**Comment by ankit-sinha-18** [comment]
Check the destination Id once again,
It seems that you have interchange your source and destination Id

**Reply by Chappie18** [reply]
**The ID was the problem.
Thank you bro for helping me to figure out the problem.
This guide helped me to find the real IDs**
https://github.com/GabrielRF/telegram-id#web-channel-id

**Comment by aahnik** [comment, marked as answer]
Are you using a bot account or user account? To forward messages to a group/channel you must be a member of it, and should be able to write messages in it using normal telegram app.

use userinfobot https://telegram.me/userinfobot to know chat ids.

**tgcf now supports using usernames/links/phone numbers in the [configuration file](https://github.com/aahnik/tgcf/wiki/How-to-configure-tgcf-%3F).**

Using chat id is no longer mandatory.
